### PR TITLE
adding a list of contributors

### PR DESCRIPTION
**Pull request by K0more**

### CODE REVIEW

**Comment by khannakshat7** [comment]
@K0more You can create a MD file instead of html for contributors such as `Contributors.md`

**Comment by K0more** [comment]
> @K0more You can create a MD file instead of html for contributors such as `Contributors.md`

alright 👍

File: Contributors.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+here is the [list of contributors](#list-of-contributors)
+# List of Contributors
+
+1. [Akshat Khanna](#akshat)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
1. [Akshat Khanna](https://www.github.com/khannakshat7)
```

File: Contributors.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+2. [Archit Khanna](#archit)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
2. [Archit Khanna](https://www.github.com/k0more) 
```

File: Contributors.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+here is the [list of contributors](#list-of-contributors)
+# List of Contributors
+
+1. [Akshat Khanna](#akshat)
+
+2. [Archit Khanna](#archit) 
+
+## Socials 
+#### Akshat
+        Twitter: @Khannakshat7
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
        Twitter: [@Khannakshat7](https://twitter.com/khannakshat7)
```

File: Contributors.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        LinkedIn: AkshatKhanna7
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
        LinkedIn: [AkshatKhanna7](https://www.linkedin.com/in/akshatkhanna7)
```

File: Contributors.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        github: khannakshat7
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
        github: [khannakshat7](https://github.com/khannakshat7)
```

File: Contributors.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+#### Archit 
+        github: k0more
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
        github: [k0more](https://github.com/k0more)
```

**Review by khannakshat7 — CHANGES_REQUESTED** [review]

File: Contributors.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+here is the [list of contributors](#list-of-contributors)
+# List of Contributors
+
+1. [Akshat Khanna](https://www.github.com/khannakshat7)
+
+2. [Archit Khanna](https://www.github.com/k0more) 
+
+## Socials 
+#### Akshat
+        Twitter: [@Khannakshat7](https://twitter.com/khannakshat7)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
- Twitter: [@Khannakshat7](https://twitter.com/khannakshat7)
```

File: Contributors.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+        LinkedIn: [AkshatKhanna7](https://www.linkedin.com/in/akshatkhanna7)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
- LinkedIn: [AkshatKhanna7](https://www.linkedin.com/in/akshatkhanna7)
```

File: Contributors.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+        github: [khannakshat7](https://github.com/khannakshat7)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
- github: [khannakshat7](https://github.com/khannakshat7)
```

File: Contributors.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+#### Archit 
+        github: [k0more](https://github.com/k0more)
```
Comment (khannakshat7) [inline Contributors.md]:
```suggestion
- github: [k0more](https://github.com/k0more)
```

**Review by khannakshat7 — CHANGES_REQUESTED** [review]

**Comment by khannakshat7** [comment]
LGTM! Great Work

**Pull request merged**